lib/modules/manager/flux/readme.md

```diff
@@ -1,7 +1,10 @@
-This manager parses [Flux](https://fluxcd.io/) YAML manifests and:
+This manager parses [Flux](https://fluxcd.io/) YAML manifests and supports:
+1. [`HelmRelease`](https://fluxcd.io/docs/components/helm/helmreleases/) resources
+2. Flux [system](https://fluxcd.io/docs/installation) manifests
```

Comment:
```suggestion
This manager parses [Flux](https://fluxcd.io/) YAML manifests and supports:

1. [`HelmRelease`](https://fluxcd.io/docs/components/helm/helmreleases/) resources
1. Flux [system](https://fluxcd.io/docs/installation) manifests

```

This should fix the linting problem.
We can just rely on the site generator to number ordered lists for us. 😉